### DISCUSSION TITLE
Bungee command execution

### DISCUSSION THREAD

**Discussion by Drc-DEV** [Zrips/CMI#2894] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
Since there's already a bungee bridge, I think it would be a nice addition to add a command to execute commands on the bungee server from the spigot server, and a way to execute a command from a spigot server to another spigot server on the network

**Comment by Baktus79** [comment]
I agree :)
